sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -153,9 +153,15 @@ case class RowDataSourceScanExec(
       pushedDownOperators.limit.map(value => "PushedLimit" -> s"LIMIT $value")
     }
 
-    Map(
-      "ReadSchema" -> requiredSchema.catalogString,
-      "PushedFilters" -> seqToString(markedFilters.toSeq)) ++
+    val filterOrPredicateInfo = if (pushedDownOperators.createdInV2) {
```

Comment:
do we really need this bool flag? I think `pushedDownOperators.pushedPredicates.nonEmpty` should do the work?

Reply:
OK